api-spec.yaml

```diff
@@ -0,0 +1,179 @@
+consumes:
```

Comment:
some quotes is needed by swagger file. 
e.g. swagger: 2.0, it is expected to be string not int.

I will remove unnecessary quotes.